src/DynamoCoreWpf/ViewModels/Core/CustomColorPickerViewModel.cs

```diff
@@ -0,0 +1,218 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Windows.Media;
+using Dynamo.Core;
+using Xceed.Wpf.Toolkit;
+
+namespace Dynamo.ViewModels
+{
+    internal class CustomColorItem : ColorItem, INotifyPropertyChanged
+    {
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        private bool isColorItemSelected = false;
+        public bool IsColorItemSelected
+        {
+            get
+            {
+                return isColorItemSelected;
+            }
+            set
+            {
+                isColorItemSelected = value;
+                OnPropertyChanged(nameof(IsColorItemSelected));
+            }
+        }
+
+        public CustomColorItem(Color? color, string name) : base(color, name)
```

Comment:
If this does nothing, do we need it?

Reply:
yes, it initialize the base class ColorItem with the parameters passed, I've added comments about it in the next commit:
https://github.com/DynamoDS/Dynamo/pull/13794/commits/a49d970a769a23938d84abc67836a231e4de443d